### DISCUSSION TITLE
Batch Error in colab

### DISCUSSION THREAD

**Discussion by archerbest** [Mikubill/sd-webui-controlnet#119] [General]
Can you help me please?

When I start the batch process, it shows me an error after generation a one image 

Error completing request
Arguments: ('task(kwnn7m308cxw57i)', 5, '((((sci-fi)))) yellow and black ((organic interior wall design shapes)) scientists ((office)) with trim hard surface metal panels and scientists in robes standing right in the middle of room, warm light atmosphere, ((encapsulated greenery)) high ceiling, highly detailed and intricate, hyper realistic, octane render, 8k Trending on Artstation, eco moody, highly detailed, unreal engine render 4k, warm light, 85mm lens', 'neg  ugly, signature, artifacts, kitchen, watermark, low quality, blurred, low resolution, bad quality, jpg artifacts, random generation, standing back to camera, woman,robot', [], None, None, None, None, None, None, None, 20, 0, 4, 0, 1, False, False, 1, 1, 7, 1.5, 0.7, 45621842.0, -1.0, 0, 0, 0, False, 192, 1280, 0, 0, 32, 0, '/content/gdrive/MyDrive/lab_ai_seq', '/content/gdrive/MyDrive/lab_ai_seq/out', '', [], 0, True, 'depth', 'control_sd15_depth [fef5e48e]', 1, None, False, 'Scale to Fit (Inner Fit)', False, False, 384, 64, 64, '<ul>\n<li><code>CFG Scale</code> should be 2 or lower.</li>\n</ul>\n', True, True, '', '', True, 50, True, 1, 0, False, 4, 1, '<p style="margin-bottom:0.75em">Recommended settings: Sampling Steps: 80-100, Sampler: Euler a, Denoising strength: 0.8</p>', 128, 8, ['left', 'right', 'up', 'down'], 1, 0.05, 128, 4, 0, ['left', 'right', 'up', 'down'], False, False, 'positive', 'comma', 0, False, False, '', '<p style="margin-bottom:0.75em">Will upscale the image by the selected scale factor; use width and height sliders to set tile size</p>', 64, 0, 2, 1, '', 0, '', 0, '', True, False, False, False, 0) {}
Traceback (most recent call last):
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/call_queue.py", line 56, in f
    res = list(func(*args, **kwargs))
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/call_queue.py", line 37, in f
    res = func(*args, **kwargs)
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/img2img.py", line 163, in img2img
    process_batch(p, img2img_batch_input_dir, img2img_batch_output_dir, img2img_batch_inpaint_mask_dir, args)
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/img2img.py", line 76, in process_batch
    processed_image.save(os.path.join(output_dir, filename))
AttributeError: 'numpy.ndarray' object has no attribute 'save'

**Comment by ljleb** [comment]
See this discussion: https://github.com/Mikubill/sd-webui-controlnet/discussions/111#discussioncomment-5000214
> In settings tab, tick the "Do not append map to output" checkbox.

**Reply by ljleb** [reply]
Note that as of 4dab9f4790d7bae517ed3269fa98593f26708add this is not necessary anymore.
